src/main/java/kg/jarkyn/server/ReceivingSocket.java

```diff
@@ -0,0 +1,22 @@
+package kg.jarkyn.server;
+
+import kg.jarkyn.server.Incoming.Requester;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+public class ReceivingSocket {
+    private ServerSocket serverSocket;
+
+    public ReceivingSocket(ServerSocket serverSocket) {
```

Comment:
I'm really not sure what the `ReceivingSocket` and `RequestingSocket` are adding? They are both just calling ServerSocket methods..